packages/components/pie-chip/src/index.ts

```diff
@@ -75,6 +78,7 @@ export class PieChip extends LitElement implements ChipProps {
                     <button
                         @click="${this._handleCloseButtonClick}"
                         ?disabled=${this.disabled}
+                        aria-label="${this.aria?.close || nothing}"
```

Comment:
I think `ifDefined` is more appropriate to use for prop values and `nothing` is when you don't want to render an element